services/core-web/src/components/mine/ExplosivesPermit/MineExplosivesPermitTable.tsx

```diff
@@ -41,6 +42,12 @@ interface MineExplosivesPermitTableProps {
   handleOpenViewExplosivesPermitModal: (event, record: IExplosivesPermit) => void;
 }
 
+type MineExplosivesColumnType = IExplosivesPermit & {
```

Comment:
I think this is the record type rather than the ColumnType? Little nitpicky, but it looks really odd to me to have `record: MineExplosivesColumnType` later on, surely we're dealing with Explosives Permits rather than listing out ColumnTypes in the table.